graphene/src/layers/vector/manipulator_group.rs

```diff
@@ -246,12 +248,12 @@ impl VectorAnchor {
 
 	/// Returns the opposing handle to the handle provided
 	/// Returns the anchor handle if the anchor is provided
-	pub fn opposing_handle(&self, handle: &VectorControlPoint) -> Option<&VectorControlPoint> {
+	pub fn opposing_handle(&self, handle: &ManipulatorPoint) -> Option<&ManipulatorPoint> {
 		self.points[!handle.manipulator_type].as_ref()
 	}
 	/// Returns the opposing handle to the handle provided, mutable
 	/// Returns the anchor handle if the anchor is provided, mutable
```

Comment:
Comments outdated (and I don't understand how this code works so I didn't try to update it myself).